### PR TITLE
dont select the full country when calculating the country code for an org

### PR DESCRIPTION
**Pull request by nicpottier**

### CODE REVIEW

**Comment by rowanseymour** [comment]
We should keep an eye on when this makes it into django https://github.com/django/django/pull/9309

**Comment by nicpottier** [comment]
Now with less footgun!

File: temba/orgs/models.py [hunk 1]
```diff
@@ -1058,7 +1058,7 @@ def find_boundary_by_name(self, name, level, parent):
             boundary = parent.children.filter(name__iexact=name, level=level)
         else:
             query = self.generate_location_query(name, level)
-            boundary = AdminBoundary.objects.filter(**query)
+            boundary = AdminBoundary.objects.filter(**query).defer('geometry', 'geometry_simplified')
```
Comment (rowanseymour) [inline temba/orgs/models.py]: these defers aren't needed now right?

File: temba/orgs/models.py [hunk 2]
```diff
@@ -1070,7 +1070,7 @@ def find_boundary_by_name(self, name, level, parent):
                 alias = BoundaryAlias.objects.filter(**query).first()
 
             if alias:
-                boundary = [alias.boundary]
+                boundary = AdminBoundary.objects.filter(id=alias.boundary_id).defer('geometry', 'geometry_simplified')
```
Comment (rowanseymour) [inline temba/orgs/models.py]: as above

**Pull request merged**